Signal/src/ViewControllers/ConversationView/Cells/OWSContactShareView.m

```diff
@@ -188,8 +188,9 @@ - (void)createContents
     [contentView autoPinEdgeToSuperviewEdge:ALEdgeTop withInset:self.vMargin];
 
     AvatarImageView *avatarView = [AvatarImageView new];
-    avatarView.image =
-        [self.contactShare getAvatarImageWithDiameter:self.iconSize contactsManager:self.contactsManager];
+    avatarView.image = [self.contactShare getAvatarImageWithDiameter:self.iconSize
+                                                     contactsManager:self.contactsManager
+                                             ignoreContactAndProfile:YES];
```

Comment:
This is the bug fix.  Previously, it would often look like you sent an avatar even if you deselected the avatar field in the "contact share approval" view, because `ContactShareViewModel.getAvatarImage()` uses `OWSContactAvatarBuilder` which tries to fill in avatars using profile and system contact avatars from the contacts manager.  

The (design) change is to have the contact share "bubbles/cards" more closely reflect the contact share's contents to avoid this (possibly troubling) confusion.